.github/workflows/lineage-management.yml

```
@@ -0,0 +1,286 @@
+          DRAFT_COUNT=$(echo "$LINEAGES_OUTPUT" | jq -r '.lineages | map(select(.status == "draft")) | length' || echo "0")
+          APPROVED_COUNT=$(echo "$LINEAGES_OUTPUT" | jq -r '.lineages | map(select(.status == "approved")) | length' || echo "0")
+          REJECTED_COUNT=$(echo "$LINEAGES_OUTPUT" | jq -r '.lineages | map(select(.status == "rejected")) | length' || echo "0")
+
+          # Enhanced validation: check for various invalid conditions
+          # 1. Empty from_ids or to_ids arrays
+          # 2. Null or non-array from_ids/to_ids
+          # 3. Invalid lineage structure
+          # Future enhancement: validate that function IDs actually exist in database
+          INVALID_REFS=$(echo "$LINEAGES_OUTPUT" | jq -r '
+            .lineages | map(select(
+              ((.from_ids // []) | length) == 0 or 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)